src/test/obj_strdup/obj_strdup.c

```diff
@@ -150,6 +152,39 @@ do_strdup_null_alloc(PMEMobjpool *pop)
 	UT_ASSERT(!TOID_IS_NULL(wcs1));
 }
 
+/*
+ * do_strdup_uint64_range -- duplicate string with
+ * type number equal to range of unsigned long long int
+ */
+static void
+do_strdup_uint64_range(PMEMobjpool *pop)
+{
+	TOID(char) str1 = do_alloc(pop, TEST_STR_1, TYPE_SIMPLE_ALLOC_1);
+	TOID(char) str2 = do_alloc(pop, TEST_STR_2, TYPE_SIMPLE_ALLOC_1);
+	TOID(char) str3 = do_alloc(pop, TEST_STR_1, TYPE_SIMPLE_ALLOC_1);
+	TOID(char) str4 = do_alloc(pop, TEST_STR_2, TYPE_SIMPLE_ALLOC_1);
+	pmemobj_strdup(pop, &str1.oid, D_RO(str2), UINT64_MAX);
```

Comment:
you are overwriting the str1 and str3 objects that were allocated using `do_alloc`.